libflux/include/influxdata/flux.h

```diff
@@ -104,7 +107,7 @@ void flux_free_stateful_analyzer(struct flux_stateful_analyzer_t *);
 
 // flux_analyze_with will analyze the ast snippet using the flux_stateful_analyzer_t and produce
 // a semantic graph for that snippet.
-struct flux_error_t *flux_analyze_with(struct flux_stateful_analyzer_t *, struct flux_ast_pkg_t *, struct flux_semantic_pkg_t **);
+struct flux_error_t *flux_analyze_with(struct flux_stateful_analyzer_t *, const char * src, struct flux_ast_pkg_t *, struct flux_semantic_pkg_t **);
```

Comment:
I see the valgrind tests are failing in CI, my guess is they need to be updated for this change and a new tests added for the new flux_error_print function.